docs/configuration/protocols/bfd.rst

```diff
@@ -9,6 +9,11 @@ BFD
 :abbr:`BFD (Bidirectional Forwarding Detection)` is described and extended by
 the following RFCs: :rfc:`5880`, :rfc:`5881` and :rfc:`5883`.
 
+In the age of very fast networks, a second of unreachability can means millions of packet loss. The idea behind BFD is to detect very quickly when a peer is down and take action extremely fast. 
```

Comment:
This is some horrible wording. ;)

Something like "In the age of very fast networks, a second of unreachability may equal millions lost packets" would be much better.